LICENSE

```diff
@@ -0,0 +1,26 @@
+Copyright (c) Audrey Roy Greenfeld and individual contributors.
```

Comment:
Would this license require users of this template to provide a copy of this license text along with the dynamically generated one in order to be compliant?